cmd/pctl/main.go

```diff
@@ -122,42 +124,109 @@ func installCmd() *cli.Command {
 				DefaultText: "profile_subscription.yaml",
 				Usage:       "Filename to use for the generated content.",
 			},
+			&cli.BoolFlag{
+				Name:  "create-pr",
+				Value: false,
+				Usage: "If given, install will create a PR for the modifications it outputs.",
+			},
+			&cli.StringFlag{
+				Name:        "remote",
+				Value:       "origin",
+				DefaultText: "origin",
+				Usage:       "The remote to push the branch to.",
+			},
+			&cli.StringFlag{
+				Name:        "base",
+				Value:       "main",
+				DefaultText: "main",
+				Usage:       "The base branch to open a PR against.",
+			},
+			&cli.StringFlag{
+				Name:  "repo",
+				Value: "",
+				Usage: "The repository to open a pr against. Format is: org/repo-name",
+			},
 		},
 		Action: func(c *cli.Context) error {
-			profilePath, catalogURL, err := parseArgs(c)
-			if err != nil {
-				_ = cli.ShowCommandHelp(c, "install")
+			// Run installation main
+			if err := runInstall(c); err != nil {
 				return err
 			}
+			// Create a pull request if desired
+			if c.Bool("create-pr") {
+				if err := runCreatePullRequest(c); err != nil {
+					return err
+				}
+			}
+			return nil
+		},
+	}
+}
 
-			branch := c.String("branch")
-			subName := c.String("subscription-name")
-			namespace := c.String("namespace")
-			configValues := c.String("config-secret")
-			filename := c.String("out")
+// runInstall runs the install part of the `install` command.
+func runInstall(c *cli.Context) error {
```

Comment:
```suggestion
func install(c *cli.Context) error {
```

Reply:
fiiine :D

Reply:
big nit sorry haha :smile:

Reply:
I renamed this, but the merge screwed me over. :D I have now this as `install` and `createPullRequest`. :)